scripts/vaccination_alerts/send-emails.ts

```diff
@@ -0,0 +1,206 @@
+      await this.db
+        .collection('invalid-alert-subscriptions')
+        .doc(email)
+        .set(currentData);
+      await this.db.collection('alerts-subscriptions').doc(email).delete();
+    }
+  }
+
+  private updateSentEmailCounters(
+    email: string,
+    alert: RegionVaccinePhaseInfo,
+  ) {
+    this.emailSent += 1;
+    this.uniqueEmailAddress[email] = (this.uniqueEmailAddress[email] || 0) + 1;
+    // Todo don't make just fips
+    const key = (alert.fips, alert.emailAlertVersion);
+    this.locationsWithEmails[key] = (this.locationsWithEmails[key] || 0) + 1;
+  }
+
+  async fetchEmailsForAlert(alert: RegionVaccinePhaseInfo) {
+    const docPath = `${alert.fips}/emailVersions/${alert.emailAlertVersion}/emails/`;
+    const querySnapshot = await this.db
+      .collection(`vaccination-alerts/${docPath}`)
+      .where('sentAt', '==', null)
+      .get();
+    return querySnapshot.docs.map(emailDoc => emailDoc.id);
+  }
+
+  async sendAlertEmail(
+    email: string,
+    alert: RegionVaccinePhaseInfo,
+    dryRun: boolean = false,
+  ): Promise<void> {
+    if (dryRun) {
+      this.updateSentEmailCounters(email, alert);
+      return;
+    }
+
+    // @ts-ignore
+    let sendData: EmailSendData = null; //  generateAlertEmailData(email, locationAlert);
+    try {
+      await this.emailService.sendEmail(sendData);
+      await this.onEmailSent(email, alert);
+    } catch (err) {
+      if (isInvalidEmailError(err)) {
+        await this.onInvalidEmail(email);
+      } else {
+        console.error(
+          `Error sending email ${email}, ${alert.fips} - ${alert.emailAlertVersion}.`,
+          err,
+        );
+        this.errorCount += 1;
+      }
+    }
+  }
+}
+
+/**
+ * Given a list of users to email and a list of locations, email those
+ * users with alert information about those locations.
+ *
+ * Takes in a file with {[fips: string]: {...data}} and file like {[fips: string]: [emails]}
+ *
+ * Also takes in a parameter to actual send the email. Otherwise will just generate the files
+ *
+ * You can run via `yarn send-emails fipsToAlertFilename currentSnapshot [send]`
+ */
+async function main(
+  alertPath: string,
+  batchSize: number,
+  dryRun: boolean,
+  singleEmail?: string,
+  fips?: string,
+) {
+  const alertsByFips = readVaccinationAlerts(alertPath);
+  const alerts = _.values(alertsByFips);
+
+  const emailService = new EmailService();
+  const db = getFirestore();
+
+  const alertEmailService = new SendVaccinationAlertsService(db, emailService);
+
+  const emailAlertTuples = _.flatten(
+    await Promise.all(
+      alerts.map(async alert => {
+        const emails = await alertEmailService.fetchEmailsForAlert(alert);
+        return (
+          emails
+            // if an email is specified, filter out all other emails. Used for testing
+            .filter(email => (singleEmail ? email === singleEmail : true))
+            .map(email => ({ email, alert }))
+        );
+      }),
+    ),
+  );
```

Comment:
The function to render and send the emails only needs the FIPS code (I just finished that), so we can probably simplify here. I think that we should remove the `singleEmail` logic to make this function easier to read